packages/webr/R/eval.R

```diff
@@ -15,26 +15,23 @@
 #' @param handlers If `TRUE`, execute the R code using a [tryCatch], with
 #' handlers in place.
 #'
+#' @useDynLib webr
```

Comment:
Should use symbol registration

```suggestion
#' @useDynLib webr, .registration = TRUE
```

Need to add an init function to register symbols.